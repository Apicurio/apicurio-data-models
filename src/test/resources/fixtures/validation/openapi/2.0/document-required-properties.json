{
    "swagger":"2.0",
    "host":"petstore.swagger.io"
}