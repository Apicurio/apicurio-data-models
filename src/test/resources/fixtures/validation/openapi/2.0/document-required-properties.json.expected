[R-002] |medium| {/->info} :: API is missing the 'info' property.
[R-003] |medium| {/->paths} :: API is missing the 'paths' property.